packages/migrate/loaders/loaders.ts

```diff
@@ -52,43 +54,43 @@ module.exports = function(j, ast) {
 	 * @returns {Node} path - object expression ast with array expression instead of loaders string
 	 */
 
-	const createArrayExpressionFromArray = function(path) {
-		const value = path.value;
+	const createArrayExpressionFromArray = (path: INode): INode => {
+		const value: INode = path.value;
 		// Find paths with `loaders` keys in the given Object
-		const paths = value.properties.filter(prop =>
-			prop.key.name.startsWith("loader")
+		const paths: INode[] = value.properties.filter((prop: INode) =>
+			prop.key.name.startsWith("loader"),
 		);
 		// For each pair of key and value
-		paths.forEach(pair => {
+		paths.forEach((pair: INode): void => {
 			// Replace 'loaders' Identifier with 'use'
 			pair.key.name = "use";
 			// If the value is an Array
 			if (pair.value.type === j.ArrayExpression.name) {
 				// replace its elements
-				const pairValue = pair.value;
+				const pairValue: INode = pair.value;
 				pair.value = j.arrayExpression(
-					pairValue.elements.map(arrElement => {
+					pairValue.elements.map((arrElement: INode): INode => {
 						// If items of the array are Strings
 						if (arrElement.type === j.Literal.name) {
 							// Replace with `{ loader: LOADER }` Object
 							return j.objectExpression([
-								utils.createProperty(j, "loader", arrElement.value)
+								utils.createProperty(j, "loader", arrElement.value),
 							]);
 						}
 						// otherwise keep the existing element
 						return arrElement;
-					})
+					}),
 				);
-				//	If the value is String of loaders like 'style!css'
+				// 	If the value is String of loaders like 'style!css'
 			} else if (pair.value.type === j.Literal.name) {
 				// Replace it with Array expression of loaders
-				const literalValue = pair.value;
+				const literalValue: INode = pair.value;
 				pair.value = j.arrayExpression(
```

Comment:
missing return?

Reply:
No. We're returning `path` at end of the block. Same as previous behavior.